src/test/java/io/kestra/plugin/aws/AbstractLocalStackTest.java

```diff
@@ -0,0 +1,32 @@
+package io.kestra.plugin.aws;
+
+import io.micronaut.test.extensions.junit5.annotation.MicronautTest;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.testcontainers.containers.localstack.LocalStackContainer;
+import org.testcontainers.junit.jupiter.Testcontainers;
+import org.testcontainers.utility.DockerImageName;
+
+@MicronautTest
+@Testcontainers
+public class AbstractLocalStackTest {
+    public static String LOCALSTACK_VERSION = "localstack/localstack:1.4.0";
+
+    protected static LocalStackContainer localstack;
+
+    @BeforeAll
+    static void startLocalstack() {
+        localstack = new LocalStackContainer(DockerImageName.parse(LOCALSTACK_VERSION))
+            .withServices(LocalStackContainer.Service.S3);
+        // some tests use a real flow with hardcoded configuration, so we have to fix the binding port
+        localstack.setPortBindings(java.util.List.of("4566:4566"));
```

Comment:
I wonder if we should'nt allow to configure the local stack before starting it (in an abstract protected method) as not all tst need this port binding and some test may need at some point other thing